install

```diff
@@ -94,8 +94,12 @@ end
 def sudo?
   return @have_sudo unless @have_sudo.nil?
 
-  Kernel.system "/usr/bin/sudo", "-v"
-  @have_sudo = $CHILD_STATUS && $CHILD_STATUS.success?
+  begin
+    Kernel.system "/usr/bin/sudo", "-l", "mkdir"
+    @have_sudo = $CHILD_STATUS.success?
```

Comment:
```suggestion
    @have_sudo = $CHILD_STATUS&.success?
```
`$CHILD_STATUS` can be `nil` if `/usr/bin/sudo` does not exist.

Reply:
Changed to 

    @have_sudo = $CHILD_STATUS && $CHILD_STATUS.success? || false

to keep compatibility for Ruby 2.2    

@have_sudo = $CHILD_STATUS&.success?
--




@have_sudo = $CHILD_STATUS&.success?
--

Reply:
No need for the `|| false`. The truthy value of `nil` for false is fine.